Alignment/OfflineValidation/interface/outTrends.h

```diff
@@ -0,0 +1,191 @@
+#ifndef OUTTRENDS_H_
```

Comment:
```suggestion
#ifndef ALIGNMENT_OFFLINEVALIDATION_OUTTRENDS_H_
```
please use full-fledged inclusion guards to minimize chances of collisions

Reply:
Done. Changed it also for remaining classes in src that did not have the guards. Let me know if this is not fine

Reply:
Looks good, thanks.